cuda/core.go

```diff
@@ -0,0 +1,30 @@
+package cuda
+
+/*
+#include <stdlib.h>
+#include "../core.h"
+#include "core.h"
+*/
+import "C"
+import (
+	"image"
+	"reflect"
+	"unsafe"
+)
+
+func toRectangles(ret C.Rects) []image.Rectangle {
```

Comment:
This is a duplicate of the code located here https://github.com/hybridgroup/gocv/blob/2d8275b67baa022d6abeda69508821f6b1db015c/core.go#L1911-L1926

Reply:
Other package,  in cuda can't visible.
If you rename to public name, cgo can't compiling.